### ISSUE TITLE
Query with array parameter doesn't work with strings that contains points

### ISSUE THREAD

**Issue by iwxfer** [envms/fluentpdo#177]
Query with array parameter doesn't works with strings that contains points:

```
$fpdo->from($table)->where("album_folder", ['aprendeingles.club', 'elemprendedorx.com']);
```

generates:

```
SELECT items.* FROM items 
LEFT JOIN aprendeingles ON aprendeingles.id = items.aprendeingles_id 
LEFT JOIN elemprendedorx ON elemprendedorx.id = items.elemprendedorx_id 
WHERE album_folder IN ('aprendeingles.club', 'elemprendedorx.com')
```

but it should generate:

```
SELECT items.* FROM items 
WHERE album_folder IN ('aprendeingles.club', 'elemprendedorx.com')
```

**Comment by cbornhoft** [comment]
With the current way Fluent handles searching for automatic joins in WHERE clauses, this isn't possible to fix. We'll keep it in mind for the 2.0 release as it's something that definitely should not happen.

In the meantime, the solution is to turn off smart join, you can then turn it back on for the next query:

    $fpdo->from($table)->disableSmartJoin()->where("album_folder", ['aprendeingles.club', 'elemprendedorx.com']);

**Comment by cbornhoft** [comment]
Fixed in 2.0
